css/sign.css

```diff
@@ -0,0 +1,123 @@
+@import url("./reset.css");
+
+body{
+    background: #F0F6FF;
+}
+
+.main-position{
+    margin: 120px 32px 233px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    max-width: 400px;
+}
+
+nav a img{
+    width: 100%;
+}
+
+form{
+    width: 100%;
+}
+
+.join-us{
+    margin-top: 16px;
+    margin-bottom: 30px;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+}
+
+.join-us a{
+    color: #6D6AFE;
+    text-decoration: underline;
+    font-weight: 600;
+    line-height: normal;
+}
+
+.input-case{
+    border-radius: 8px;
+    padding: 18px 15px;
+    width: 100%;
+    height: auto;
+    border: 1px solid #CCD5E3;
+    margin-bottom: 24px;
+    }
+
+.input-case:focus{
+    border: 1px solid #6D6AFE;
+}
+
+.h-12 {
+    width: 100%;
+    height: 12px;
+    margin-bottom: 0;
+}
+
+.sign-title{
+    font-size: 14px;
+    font-weight: 400;
+}
+
+.signin-button{
+    color: #F5F5F5;
+    font-size: 18px;
+    font-weight: 600;
+    border-radius: 8px;
+    background: var(--gra-purpleblue-to-skyblue, linear-gradient(91deg, #6D6AFE 0.12%, #6AE3FE 101.84%));
+    border-radius: 8px;
+    padding: 16px 20px;
+    margin-top: 6px;
+    width: 100%;
+}
+
+.login-footer{
+    border-radius: 8px;
+    border: 1px solid #CCD5E3;
+    background: #E7EFFB;
+    width: 100%;
+    height: auto;
+    margin-top: 32px;
+    padding: 12px 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.social-login{
+    font-size: 14px;
+    font-weight: 400;
+    color: #373740;
+}
+
+.login-footer a:first-child{
+    margin-right: 16px;    
+}
+
+.login-footer div a img{
+    width: 42px;
+}
+
+.eye {
+    width: 16px;
+    height: 16px;
+    position: absolute;
+    right: 15px;
+    bottom: 43px;
+}
```

Comment:
이 경우에 bottom:43px과 같이 별도의 px값으로 세로축의 위치를 맞춰주면 만약에 input의 높이가 달라진다는 가정하에 아이콘의 위치가 깨질 수 있는데요~
 
html의 구조가 아래와 같다는 가정하에는 (현재 승재님이 작성해주신 구조와 동일합니다) <div>에 position: relative를 주고 .eye class에는 top: 50%, transform: translateY(-50%) 이렇게 해주시면 높이에 맞게 딱 중간에 위치하게 됩니다!

```html
<div>
  <input />
  <button>
    <img />
  </button>
</div> 
```
```suggestion
.eye {
    width: 16px;
    height: 16px;
    position: absolute;
    top: 50%;
    right: 15px;
    transform: translateY(-50%);
}
```